src/Native/build-native.proj

```diff
@@ -19,8 +19,9 @@
       <_ProcessorCountArg> --numproc $(MSBuildNodeCount)</_ProcessorCountArg>
       <_StripSymbolsArg Condition="'$(BuildNativeStripSymbols)' == 'true'"> stripsymbols</_StripSymbolsArg>
       <_PortableBuildArg Condition="'$(PortableBuild)' == 'true'"> -portable</_PortableBuildArg>
+      <_ClangBuildArg Condition="'$(Clang)' != ''"> $(Clang)</_ClangBuildArg>
```

Comment:
The change looks good but I think we need a little more specific property to avoid potential conflicts. Can you call it something like `BuildNativeClang` or something similar.